meson.build

```diff
@@ -0,0 +1,50 @@
+project('pyseldon', 'cpp', 
+    default_options: ['cpp_std=c++20', 'warning_level=3'])
+
+_args = ['-fPIC']
```

Comment:
`pic: True` should be used instead of manually passing `-fPIC`.
https://mesonbuild.com/Reference-manual_functions.html#library_pic

Reply:
pic is required for the shared lib and it is set in the Seldon codebase, here it is used for c++ compiler to know that the code should be pic